SU-HSDO/suhumsci
# SHS-5515: Refactor vertical card main link code

# READY FOR REVIEW

## Summary
- Refactor code for handing vertical card main link, as requested in FK Release 47 PR
- Fix issue with special characters in stage reported by @ahughes3

**NOTE:** Special character issue was being caused by DOMElement `saveHTML()` encoding error. Removing DOMElement automatically fixes the issue.

## Need Review By (Date)
- 04/17

## Urgency
high

## Steps to Test
1. Install `english` site
2. Visit https://english.suhumsci.loc/people/faculty
3. Confirm that you can click anywhere on the card and you'll be redirected to the person profile
4. Confirm that names containing special characters (e.g. Ramón Saldívar) are being displayed correctly

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
